explore/src/main/scala/explore/constraints/ConstraintsPanel.scala

```diff
@@ -194,76 +191,49 @@ object ConstraintsPanel {
+                                                                   Icons.Question.fitted(true)
               ),
-              ".",
-              Button(
-                compact = true,
-                size = Small,
-                basic = true,
-                circular = true,
-                inverted = true,
-                floated = floats.Right,
-                clazz = ExploreStyles.ButtonDismiss,
-                onClick = state.zoom(State.showMultiEditWarning).set(none).runAsyncCB
-              )("Dismiss")
-            )
+            content = PopupContent(
+              content = <.div(ExploreStyles.SharedEditWarning)(
+                s"These constraints cannot be modified because they are shared by $obsCount observations.",
+                <.br,
+                "Please edit in Constraints View, create a New one or a Copy."
+              )
+            ),
+            wide = PopupWide.Very,
+            position = PopupPosition.TopCenter
           )
-          .when(props.multiEditWarnings)
+        ).when(!props.allowMultiEdit)
 
       <.div(
-        ^.onFocus ==> (_ => showWarningView.set(obsCount > 1).runAsyncCB),
-        ^.onComponentBlur(showWarningView.set(false).runAsyncCB)
-      )(
-        editWarning,
         props.renderInTitle(
-          <.span(
-            <.span(ExploreStyles.TitleUndoButtons, UndoButtons(constraintSet.get, undoCtx)),
-            <.span(ExploreStyles.TitleButtonStrip)(
-              Popup(
-                trigger = Button(onClick = makeCopy.runAsyncCB,
-                                 size = Small,
-                                 compact = true,
-                                 disabled = state.get.copying
-                )(Icons.Copy.fitted(true)),
-                content = "Copy"
-              ).when(props.copyButton)
-            )
+          <.span(ExploreStyles.TitleStrip)(
+            editWarning.when(obsCount > 1),
+            Button(
+              onClick = makeCopy.runAsyncCB,
+              size = Tiny,
+              compact = true,
+              disabled = state.get.copying,
+              icon = Icons.Copy,
+              content = "Copy"
+            ),
+            UndoButtons(constraintSet.get, undoCtx)
           )
         ),
-        Form(as = <.div, loading = state.get.copying)(ExploreStyles.Grid,
-                                                      ExploreStyles.ConstraintsGrid
-        )(
+        Form(loading = state.get.copying)(ExploreStyles.Grid, ExploreStyles.ConstraintsGrid)(
           FormInputEV(
             id = "name",
             label = Label("Name", HelpIcon("constraints/main/name.md")),
             clazz = ExploreStyles.ConstraintsNameField,
             value = nameView,
             validFormat = ValidFormatInput.nonEmptyValidFormat,
             errorClazz = ExploreStyles.InputErrorTooltip,
-            errorPointing = LabelPointing.Below
+            errorPointing = LabelPointing.Below,
+            disabled = readonly
```

Comment:
I wonder if you can disable the whole form in one go, I suspect no

Reply:
I investigated that initially, but I couldn't find a way.